scripts/launch_cluster.py

```diff
@@ -0,0 +1,640 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+#
+# Create a Kubernetes cluster on AWS/Aliyun using python sdk.  
+#
+# Notice: this script would require your AWS/Aliyun account's access_key_id and secret_access_key
+# to get access your clusters' information or create a Kubernetes cluster automatically, finally output
+# a kube config of cluster. 
+# 
+# This script assumes that kubectl and aws-iam-authenticator are in your PATH.
```

Comment:
This should be checked before launcher starts. If not exists, remind user to install them.

Reply:
done.